Tests.Integration.Presentation.Web/Contract/V2/ItContractsApiV2Test.cs

```diff
@@ -546,7 +676,7 @@ private string CreateEmail()
             return $"{nameof(ItContractsApiV2Test)}{A<string>()}@test.dk";
         }
 
-        private static void AssertCrossReference<T>(T expected, IdentityNamePairResponseDTO actual) where T : IHasNameExternal, IHasUuidExternal
+        private static void AssertCrossReference<TExpected, TActual>(TExpected expected, TActual actual) where TExpected : IHasNameExternal, IHasUuidExternal where TActual : IHasNameExternal, IHasUuidExternal
```

Comment:
Nogen grund til to generiske typer når de indeholder begge dele? Udover at øge læsbarheden

Reply:
Det gør det muligt at komme med to forskellige lhs og rhs typer så længe de samme type constraints overholdes